### DISCUSSION TITLE
Votive with the v4.0 nuget package

### DISCUSSION THREAD

**Discussion by JohnLudlow** [wixtoolset/issues#6553] [Questions] [answered]
We'll be migrating an installer from InstallShield to WiX shortly, so I wondered if this was the best point to investigate WiX 4.

I created a test setup project with the 2019 extension and the 3.11.2 nuget package. That worked so I tried updating to the v4.0 package. This is the error I get 

```
Restoring packages for C:\repos\ConsoleApp3\ConsoleApp3\ConsoleApp3.csproj...
NU1202: Package wix 4.0.0-preview.0 is not compatible with net5.0 (.NETCoreApp,Version=v5.0). Package wix 4.0.0-preview.0 supports: netcoreapp3.1 (.NETCoreApp,Version=v3.1) / any
NU1212: Invalid project-package combination for wix 4.0.0-preview.0. DotnetToolReference project style can only contain references of the DotnetTool type 
Package 'wix 4.0.0-preview.0' has a package type 'DotnetTool' that is not supported by project 'ConsoleApp3'.
Time Elapsed: 00:00:00.2313980
========== Finished ==========
```

Do I need to create a different project type for v4.0? Is there a good way to convert between them? Should I be using a different version of the extension? A different extension?

Or is the extension for v4.0 just not quite ready yet?

(I did consider raising an issue but been out of WiX for a while so it's just as likely I don't know what's going on these days. I'll raise an issue if that makes sense based on the discussion)

**Comment by robmen** [comment, marked as answer]
WiX v4.0-preview.0 does not include MSBuild support (only the command-line tools), see: https://www.firegiant.com/blog/2021/5/27/wix-online-meeting-213-highlights/ and https://robmensching.com/blog/posts/2021/5/17/wix-toolset-v4-preview.0/

**Reply by JohnLudlow** [reply]
Ah so it falls under the "extension's not quite ready yet" category.

Fair enough.  Thanks
